### ISSUE TITLE
Remove black focus rectangle from inputs except after keyboard input

### ISSUE THREAD

**Issue by owenssww** [microsoft/fast-dna#1746]
Currently, the input field behavior calls for a 2px black stroke to appear in the active state. For non-keyboard users, this focus is a little too severe. 

My suggested improvement to this behavior is to remove the 2px black stroke if the active state is triggered by mouse input, and instead use a lighter treatment, for example a 1px stroke using the accent base color. If and only if the active state is triggered by keyboard input, then the 2px black border would be triggered.

<img width="900" alt="Text input suggestions" src="https://user-images.githubusercontent.com/50466548/57489860-343da380-726c-11e9-862e-059eed6f488c.png">
